include/alpaka/core/Debug.hpp

```diff
@@ -146,36 +146,3 @@ namespace alpaka
 #else
     #define ALPAKA_DEBUG_BREAK
 #endif
-
-//-----------------------------------------------------------------------------
```

Comment:
This changes should not be a part of this pull request. Please move this changes to a new pull request.